madara/crates/client/rpc/src/versions/user/v0_7_1/methods/read/syncing.rs

```diff
@@ -16,30 +16,63 @@ use crate::Starknet;
 /// * `Syncing` - An Enum that can either be a `mc_rpc_core::SyncStatus` struct representing the
 ///   sync status, or a `Boolean` (`false`) indicating that the node is not currently synchronizing.
 pub async fn syncing(starknet: &Starknet) -> StarknetRpcResult<SyncingStatus> {
```

Comment:
Can we change the fn's name for a more descriptive one (get_status/is_syncing) ? Also I think the description needs to be updated.

Reply:
@Mohiiit wdyt?